http/handler.go

```diff
@@ -172,6 +172,7 @@ func handler(props *vault.HandlerProperties) http.Handler {
 		mux.Handle("/v1/sys/rekey-recovery-key/init", handleRequestForwarding(core, handleSysRekeyInit(core, true)))
 		mux.Handle("/v1/sys/rekey-recovery-key/update", handleRequestForwarding(core, handleSysRekeyUpdate(core, true)))
 		mux.Handle("/v1/sys/rekey-recovery-key/verify", handleRequestForwarding(core, handleSysRekeyVerify(core, true)))
+		mux.Handle("/v1/sys/init-unseal-recovery", handleRequestForwarding(core, handleSysInitUnsealRecovery(core)))
```

Comment:
```suggestion
		mux.Handle("/v1/sys/enable-unseal-recovery", handleRequestForwarding(core, handleSysInitUnsealRecovery(core)))
```

I think the associations of your proposed endpoint with sys/init make sense on one level, but are inappropriate on another, i.e. it implies that this must be done on an un-initialized cluster.